cloud/os_floating_ip.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# Copyright (c) 2014 Hewlett-Packard Development Company, L.P.
+# Copyright (c) 2013, Benno Joy <benno@ansible.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+import time
+
+try:
+    import shade
+except ImportError:
+    print("failed=True msg='shade is required'")
+
+DOCUMENTATION = '''
+---
+module: os_floating_ip
+short_description: Add/Remove floating IP from an instance
```

Comment:
I'm merging os_compute_floating_ip into this one. The os_compute module lets you set auto_floating_ip to automatically get an IP at server create time. I'm honestly not sure how useful this module will be anyway, other than to maybe remove a floating IP. But it could be enhanced in a future version to accept a known IP.